tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -295,6 +295,7 @@ cc_library(
         "@llvm//:object",
         "@llvm//:support",
         "@llvm//:target",
+        "@llvm//:polly",
```

Comment:
Can we only conditionally pull in this dependency?  Not everyone will use polly, so it makes sense to avoid pulling in the dependency for folks who don't need it.